qatrfm/utils/logger.py

```diff
@@ -62,7 +63,11 @@ def warning(self, msg, *args, **kwargs):
     @staticmethod
     def getQatrfmLogger(name):
         logging.setLoggerClass(QaTrfmLogger)
-        return logging.getLogger(name)
+        handler = logging.StreamHandler(sys.stdout)
+        handler.setLevel(logging.DEBUG)
+        logger = logging.getLogger(name)
+        logger.addHandler(handler)
+        return logger
```

Comment:
to make it more complete, we can also add a formatter, for example:

> formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
> handler.setFormatter(formatter)

Reply:
done